src/coffee/run.coffee

```diff
@@ -35,7 +35,7 @@ argv = require('optimist')
   .describe('logDir', 'directory to store logs')
   .describe('logSilent', 'use console to print messages')
   .describe('timeout', 'Set timeout for requests')
-  .default('csvHeaders', 'sku, quantity')
+  .default('csvHeaders', 'sku, quantityOnStock')
```

Comment:
Btw: this option is used to pass custom header fields. Are we still going to support this or the headers will be fixed?